File: C/2019/file5.c
// file5.c
#include <stdio.h>

int main(void) {
  char *filename = "file2.txt";

  FILE *fp = fopen(filename, "r");
  if (fp == NULL) {
    printf("ファイル %s は存在しません\n", filename);
    return -1;
  }
  //
  printf("ファイル %s をopenしました。\n", filename);

  // 読み込み
  char buf[10];
  fgets(buf, 10, fp);
  printf("%s", buf);

  //
  fclose(fp);
  //
  return 0;
}

File: PhpBeginner/2021/EarlyPeriod(Tue.)/file_5.php
<?php   // file_5.php

$dir = __DIR__ . '/file_write';
var_dump( is_writable( $dir ) );

//
$file_name = $dir . '/' . date('Ymd') . '.txt';
$log = date('Y-m-d H:i:s') . "\t test log \n";
file_put_contents($file_name, $log, FILE_APPEND | LOCK_EX);

/*
$file_name = $dir . '/' . time() . '.txt';
$r = file_put_contents($file_name, 'text body');
var_dump($r);
*/

/*
fopen()
fputs / fwrite
fclose
*/